scripts/ci/sync_extensions.py

```diff
@@ -119,25 +126,33 @@ def main():
     target_index = DEFAULT_TARGET_INDEX_URL
     os.mkdir(os.path.join(temp_dir, 'target'))
     target_index_path = os.path.join(temp_dir, 'target', 'index.json')
-    download_file(target_index, target_index_path)
-
+    try:
+        download_file(target_index, target_index_path)
+    except Exception as ex:
```

Comment:
Too broad exception.

Reply:
As it's not customer facing, we can just fail and retry later.